styles.css

```diff
@@ -0,0 +1,349 @@
+}
+
+.salutation {
+  font-size: 40px;
+  font-weight: 700;
+  width: auto;
+  color: #172b4d;
+  line-height: 52px;
+  padding-right: 20px;
+}
+
+.lets-connect {
+  font-size: 16px;
+  font-weight: 500;
+  color: #7f8cff;
+  margin-top: 10px;
+}
+
+.bars {
+  padding-right: 22px;
+}
+
+.logo {
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  padding-left: 22px;
+  line-height: 20px;
+  color: #6070ff;
+  background-color: #fff;
+}
+
+.buttons {
+  padding-bottom: 100px;
+}
+
+.buttons li {
+  display: inline-block;
+  margin-right: 20px;
+  margin-top: 25px;
+  margin-left: 0%;
+}
+
+.buttons li a {
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.menu-icon {
+  cursor: pointer;
+}
+
+.nav-bar {
+  color: #7f8cff;
+}
+
+.paragraph1 {
+  margin: 10px 0;
+  width: auto;
+  margin-left: 0;
+}
+
+.first-section {
+  background-image: url('./images/background.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  background-color: #fff;
+  margin-bottom: 50px;
+  padding-top: 50px;
+  border: 1px solid white;
+  border-bottom-left-radius: 110px;
+}
+
+.work-section {
+  display: -ms-grid;
+  display: grid;
+  width: 100%;
+  gap: 40px;
+  place-items: center;
+  margin-bottom: 58px;
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  padding: 16px;
+  margin: 16px;
+  border-radius: 16px;
+}
+
+.project-card img {
+  margin-left: 0;
+}
+
+.head {
+  color: #172b4d;
+  font-weight: bold;
+  line-height: 44px;
+  font-size: 35px;
+  font-family: poppins, sans-serif;
+  margin-left: 5px;
+  padding-top: 12px;
+}
+
+.card-detail {
+  display: inline;
+  text-align: center;
+}
+
+.card-detail ::after {
+  content: "\00b7";
+  font-size: 1.5em;
+}
+
+.card-detail :last-child::after {
+  content: none;
+}
+
+.langu {
+  background-color: #ebebff;
+  padding: 10px;
+  color: #4472e7;
+  border-radius: 10px;
+  font-weight: 500;
+  font-family: poppins, sans-serif;
+  display: inline-block;
+  margin-left: 10px;
+}
+
+.circle {
+  display: inline-block;
+  flex-direction: row;
+  align-items: center;
+  padding: 1px 5px;
+  row-gap: 12px;
+}
+
+.option {
+  font-family: poppins, sans-serif;
+  color: #7a869a;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  position: static;
+  width: 88px;
+  padding-top: 8px;
+}
+
+.canopy {
+  font-family: poppins, sans-serif;
+  color: #344563;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  position: static;
+  width: 55px;
+  height: 16px;
+  padding-left: 0;
+  padding-top: 8px;
+  margin-left: 10px;
+}
+
+p {
+  position: static;
+  width: auto;
+  height: auto;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  order: 1;
+  margin: 12px 10px;
+}
+
+button {
+  border: 2px solid #4472e7;
+  padding: 8px;
+  color: #4472e7;
+  text-decoration: none;
+  border-radius: 8px;
+  font-size: large;
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  background-color: #fff;
+}

```

Comment:
- [ ] Please the border size `1px` instead of `2px` like the Figma design.
- [ ] And Kindly edit the font weight also to be `500` to match the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170727792-7cb6b307-f4ac-42d3-a0fa-3f2503a16a6d.png)  | ![image](https://user-images.githubusercontent.com/68796232/170727771-a6e628e1-3f37-45bd-ba4d-e22a5b95692c.png) |

### By the way Great work on the styling 💪.